src/Cake.Svn.Tests/Unit/Add/SvnAddSettingsExtensionsTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using Cake.Svn.Add;
+using Xunit;
+
+namespace Cake.Svn.Tests.Unit.Add
+{
+    public sealed class SvnAddSettingsExtensionsTests
+    {
+        public sealed class TheConstructor
+        {
+            [Fact]
+            public void Should_Throw_If_AddSettings_Is_Null()
```

Comment:
Parameter is named `settings` so `Should_Throw_If_Settings_Is_Null` would be a better fit